scaling_performance/optimizing_storage.adoc

```diff
@@ -11,19 +11,167 @@
+
+.Available storage options
+[options="header"]
+|===
+| Storage type | Description | Examples
+
+| Block
+| This type of storage is presented to the operating system as a block device. It is
+suitable for applications that need full control of storage and operate at a low level on files
+bypassing the file system. It is sometimes also referred to as Storage Area Network (SAN). It is
+usually a non-shareable type of storage, which means that only one client at a time can
+mount an endpoint of this type.
+| CNS/CRS GlusterFS footnoteref:[disclaimer, CNS/CRS GlusterFS, Ceph RBD, OpenStack Cinder, AWS EBS, Azure Disk, GCE persistent disk, and VMware vSphere support dynamic PV provisioning natively in OpenShift Container Platform.] , iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, AWS EBS footnoteref:[disclaimer], Dell/EMC Scale.IO,
+VMware vSphere Volume, GCE Persistent Disk footnoteref:[disclaimer], Azure Disk
+
+| File
+| This type of storage is presented to the operating system as a file system export to be mounted. File storage is
+sometimes also referred to as Network Attached Storage (NAS). Concurrency, latency, file locking mechanisms and other
+capabilities will vary widely between protocols, implementations, vendors, and scales.
+| CNS/CRS GlusterFS footnoteref:[disclaimer], RHEL NFS, NetApp NFS footnoteref:[disclaimer2, NetApp NFS supports Dynamic PV provisioning when using the Trident plugin.], Azure File, Vendor NFS, Vendor GlusterFS footnoteref:[disclaimer3, Vendor GlusterFS, Vendor S3, and Vendor Swift supportability and configurability may vary.], Azure File, AWS EFS
+
+| Object
+| This type of is storage is accessible via a REST API endpoint. Object storage is configurable for use in the {product-title} Registry. Applications
+intending to use this storage must build their drivers into the application and/or container.
+| CNS/CRS GlusterFS footnoteref:[disclaimer], Ceph Object Storage (RADOS Gateway), OpenStack Swift, Aliyun OSS, AWS S3, Google Cloud Storage, Azure Blob Storage, Vendor S3 footnoteref:[disclaimer3], Vendor Swift footnoteref:[disclaimer3]
+|===
+
+[NOTE]
+====
+As of {product-title} 3.6.1, Container-Native Storage (CNS) GlusterFS (a hyperconverged or cluster-hosted storage solution) and Container-Ready Storage (CRS)
+GlusterFS (an externally hosted storage solution) provides interfaces for block, file, and object storage for the purpose of the {product-title} registry, logging, and metrics.
+====
+
+[[back-end-recommendations]]
+=== Storage Back-end Recommendations
+
+The following table summarizes the recommended and configurable storage back-end types for the given {product-title} cluster application.
+
+.Recommended and configurable storage back-end types
+[options="header"]
+|===
+|Storage type |ROX |RWX |Registry|Scaled registry| Metrics|Logging|Apps
+
+| Block
+| Yes footnoteref:[disclaimer4, This does not apply to physical disk, VM physical disk, VMDK, loopback over NFS, AWS EBS, and Azure Disk.]
```

Comment:
Usually I prefer `✅` for Yes and `🞩` for No. Up to you if you want to change that. :innocent: